selfdrive/can/process_dbc.py

```diff
@@ -38,48 +38,64 @@ def main():
     if dbc_mtime < out_mtime and template_mtime < out_mtime and this_file_mtime < out_mtime:
       continue #skip output is newer than template and dbc
 
-    msgs = [(address, msg_name, msg_size, sorted(msg_sigs, key=lambda s: s.name not in ("COUNTER", "CHECKSUM"))) # process counter and checksums first
+    msgs = [(address, msg_name, msg_size, sorted(msg_sigs, key=lambda s: s.name not in ("COUNTER", "CHECKSUM", "CRC"))) # process counter and checksums first
             for address, ((msg_name, msg_size), msg_sigs) in sorted(can_dbc.msgs.items()) if msg_sigs]
 
     def_vals = {a: set(b) for a,b in can_dbc.def_vals.items()} #remove duplicates
     def_vals = [(address, sig) for address, sig in sorted(def_vals.items())]
 
     if can_dbc.name.startswith("honda") or can_dbc.name.startswith("acura"):
-      checksum_type = "honda"
+      car_type = "honda"
       checksum_size = 4
+      counter_size = 2
     elif can_dbc.name.startswith("toyota") or can_dbc.name.startswith("lexus"):
-      checksum_type = "toyota"
+      car_type = "toyota"
       checksum_size = 8
+      counter_size = None
+    elif can_dbc.name.startswith("vw") or can_dbc.name.startswith("volkswagen") or can_dbc.name.startswith("audi") or can_dbc.name.startswith ("seat") or can_dbc.name.startswith("skoda"):
+      car_type = "volkswagen"
+      checksum_size = 8
+      counter_size = 4
     else:
-      checksum_type = None
+      car_type = None
+      checksum_size = None
+      counter_size = None
```

Comment:
I like this generalization. What about defining start bits for both counter and checksum as well? Basically after this line there should be no reference to `car_type`, except for the call to `template.render`.